lib/converter.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const { currentDate } = require('./current-date');
+
+const parse = date => {
+  const splitted = date.split('');
+  const separator = [splitted.findIndex(e => isNaN(e))];
+  return date.split(date[separator]);
```

Comment:
Passing an array to array-subscript operator is not really a good idea, it would just be `.toString()` and used to access an appropriate property.